libvirt-formula/pillar.example

```diff
@@ -24,3 +24,13 @@ libvirt:
     tcp: true
     # admin -> libvirtd-admin.socket, ro -> libvirtd-ro.socket
     # libvirtd -> libvirtd.socket (will not get 'libvirtd-' prepended, unlike the other examples)
+
+  # contrary the examples above, the following does not reflect the formula defaults. by default, none of the below is configured.
+  # everything under "guests" will be set in /etc/sysconfig/libvirt-guests - except for "enable", which defines whether the service should be enabled.
```

Comment:
Didn't we have a similar layout somewhere else, and switched it to a `config:` sub-key after I complained? ;-)

Reply:
I wanted it to align with the existing keys in this pillar, which are not further nested either.

Reply:
Ok, then let's hope we don't need more "special" options later ;-)